airbyte-webapp/src/components/Table/Table.tsx

```diff
@@ -33,6 +33,7 @@ const TableView = styled(Card).attrs({ as: "table" })<{ light?: boolean }>`
   width: 100%;
   max-width: 100%;
   border-radius: 10px;
+  overflow: hidden;
```

Comment:
What happens when this overflow is hidden and the screen size is small?

Reply:
Oops!  This should have been removed, thanks.  Was part of trying to get the sticky headers to work.  I've split that into its own issue #16435 .